index.html

```diff
@@ -69,9 +73,10 @@ <h2 class="lets-connect">LET’S CONNECT</h2>
       <!-- Work Section -->
       <section class="work-section">
         <div class="image1">
-          <img src="images/snapshot.png" alt="snapshot-image">
+          <img class="mobile" src="./images/snapshot.png" alt="snapshot-image">
+          <img class="img1 desktop" src="./images/snapshot-2.png" alt="snapshot4-image">
         </div>
-        <section class="left-block">
+        <div class="left-block">
           <h2 class="title"> Tonic </h2>
```

Comment:
Please, modify the margins and paddings around the project cards to match the design. Check the comment above for more on why you should stick to a design when given.

Your design:
![image](https://user-images.githubusercontent.com/20554583/186800897-787af9c8-3980-44d1-b245-38eec897a4f3.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/186800965-eb488318-9471-4aef-b9d6-eac1cc1d49f5.png)